src/katello/constants.py

```diff
@@ -9,3 +9,6 @@
 DISABLE_PACKAGE_PROFILE_VAR = 'DISABLE_KATELLO_PACKAGE_PROFILE'
 DISABLE_ENABLE_REPOS_VAR = 'DISABLE_KATELLO_ENABLED_REPOS'
 PROFILE_CACHE_FILE = '/var/lib/rhsm/cache/profile.json'
+
+TOOL_YUM = 1
```

Comment:
What if we made these booleans?

Something like:

```
YUM = 'yum' in sys.modules
ZYPPER = 'zypp_plugin' in sys.modules
```

I'm not sure if that's better than checking existence of the repos file, but that could be another option to make them booleans.

The reason I suggest this is because we can use this in bin/katello-enabled-repos-upload *and* we can avoid passing TOOL_YUM/TOOL_ZYPPER through the constructor. We can just let EnabledReport import those and check for itself.